.eslintrc.js

```diff
@@ -45,6 +45,7 @@ module.exports = {
     'arrow-spacing': [ 'error', { before: true, after: true } ],
     'block-spacing': 'error',
     'brace-style': [ 'error', '1tbs', { allowSingleLine: true } ],
-    'comma-style': 'error'
+    'comma-style': 'error',
+    "@typescript-eslint/no-unused-vars": [ "error", { "argsIgnorePattern": "^_", "varsIgnorePattern": "^_" } ]
```

Comment:
Sort in the alphabet, this line should be put https://github.com/alibaba/pipcook/blob/d1923a140fe84f6e773c4947b0a8489bef030a86/.eslintrc.js#L24.